src/interceptorHelperFunctions.js

```diff
@@ -13,35 +13,6 @@ function mergeObjRecursive(obj1, obj2) {
   return obj3;
 }
 
-if (Array.prototype.equals)
-// attach the .equals method to Array's prototype to call it on any array
-{
-  Array.prototype.equals = function(array) {
-    // if the other array is a falsy value, return
-    if (!array) {
-      return false;
-    }
-
-    // compare lengths - can save a lot of time
-    if (this.length !== array.length) {
-      return false;
-    }
-
-    for (let i = 0, l = this.length; i < l; i++) {
-      // Check if we have nested arrays
-      if (this[i] instanceof Array && array[i] instanceof Array) {
-        // recurse into the nested arrays
-        if (!this[i].equals(array[i])) {
-          return false;
-        }
-      } else if (this[i] !== array[i]) {
-        // Warning - two different object instances will never be equal: {x:20} != {x:20}
-        return false;
-      }
-    }
-    return true;
-  }
-}
 // Hide method from for-in loops
```

Comment:
@spiray  - I believe this can be removed as well. Apart from that, looks good!

Reply:
Removed in the next commit, thanks